docs/getting_started.md

```diff
@@ -166,14 +166,26 @@ path that it ran (i.e. `out/kubeconfig`). This is the **admin** kubeconfig file
 going forward to spin up multiple clusters using Cluster API, however, it is recommended that you create dedicated roles
 with limited access before doing so.
 
+Note that from this point forward, you no longer need to use `clusterctl` to provision clusters since your management cluster
+(the cluster used to manage clusters) has been created. Clusters should be provisioned by applying Cluster API resources
```

Comment:
(the cluster used to manage _workload_ clusters) has been created. Workload clusters...